apis/compute/v1alpha1/serverset_types.go

```diff
@@ -162,7 +162,7 @@ type ServerSetBootVolumeMetadata struct {
 }
 
 // ServerSetBootVolumeSpec are the configurable fields of a ServerSetBootVolumeSpec.
-// +kubebuilder:validation:XValidation :rule="hasOneOf", message="One of `imagePassword` or `sshKeys` must be set"
+// +kubebuilder:validation:XValidation:rule="has(self.imagePassword)  || has(self.sshKeys) ",message="either imagePassword or sshKeys must be set"
```

Comment:
extra whitespaces in the rule definition
`rule="has(self.imagePassword)  || has(self.sshKeys) "`

Reply:
```suggestion
// +kubebuilder:validation:XValidation:rule="has(self.imagePassword) || has(self.sshKeys)",message="either imagePassword or sshKeys must be set"
```